docs/source/reference/manipulation.rst

```diff
@@ -4,76 +4,116 @@ Array Manipulation Routines
 Basic manipulations
 -------------------
 
-.. autofunction:: cupy.copyto
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   cupy.copyto
 
 
 Shape manipulation
 ------------------
 
-.. autofunction:: cupy.reshape
-.. autofunction:: cupy.ravel
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   cupy.reshape
+   cupy.ravel
 
 
 Transposition
 -------------
 
-.. autofunction:: cupy.rollaxis
-.. autofunction:: cupy.swapaxes
-.. autofunction:: cupy.transpose
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   cupy.rollaxis
+   cupy.swapaxes
+   cupy.transpose
 
 
 Edit dimensionalities
 ---------------------
 
-.. autofunction:: cupy.atleast_1d
-.. autofunction:: cupy.atleast_2d
-.. autofunction:: cupy.atleast_3d
-.. autoclass:: cupy.broadcast
-.. autofunction:: cupy.broadcast_arrays
-.. autofunction:: cupy.broadcast_to
-.. autofunction:: cupy.expand_dims
-.. autofunction:: cupy.squeeze
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   cupy.atleast_1d
+   cupy.atleast_2d
+   cupy.atleast_3d
+   cupy.broadcast
+   cupy.broadcast_arrays
+   cupy.broadcast_to
+   cupy.expand_dims
+   cupy.squeeze
 
 
 Changing kind of array
 ----------------------
 
-.. autofunction:: cupy.asfortranarray
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   cupy.asarray
+   cupy.asanyarray
+   cupy.asfortranarray
+   cupy.ascontiguousarray
```

Comment:
Yes, because they appear in both [Array creation routines](https://docs.scipy.org/doc/numpy/reference/routines.array-creation.html) and [Array manipulation routines](https://docs.scipy.org/doc/numpy/reference/routines.array-manipulation.html) in NumPy API reference.